go.mod

```diff
@@ -22,9 +22,9 @@ require (
 	github.com/cosmos/cosmos-proto v1.0.0-beta.5
 	github.com/cosmos/cosmos-sdk v0.50.9
 	github.com/cosmos/gogoproto v1.7.0
-	github.com/cosmos/ibc-go/modules/apps/callbacks v0.0.0-20250227161037-91dda0165967 // v8.6.0
+	github.com/cosmos/ibc-go/modules/apps/callbacks v0.0.0-20250227195741-59987d52d959 // v8.6.1
 	github.com/cosmos/ibc-go/modules/capability v1.0.1
-	github.com/cosmos/ibc-go/v8 v8.6.0
+	github.com/cosmos/ibc-go/v8 v8.6.1
```

Comment:
it's also a technically breaking change, right?

Reply:
yes